server/controllers/websocket/writer.go

```diff
@@ -0,0 +1,70 @@
+package websocket
+
+import (
+	"net/http"
+
+	"github.com/gorilla/websocket"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/logging"
+)
+
+func NewWriter(log logging.SimpleLogging) *Writer {
+	upgrader := websocket.Upgrader{}
+	upgrader.CheckOrigin = func(r *http.Request) bool { return true }
+	return &Writer{
+		upgrader: upgrader,
+		log:      log,
+	}
+}
+
+type Writer struct {
+	upgrader websocket.Upgrader
+
+	//TODO: Remove dependency on atlantis logger here if we upstream this.
```

Comment:
nit: Not sure if this comment is still relevant.

Reply:
Nope! Thanks for the catch :)